examples/solidjs-live-avatars/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "solidjs-live-avatars",
+  "description": "This example shows how to build a live avatar stack with Liveblocks and Solid.js",
+  "license": "Apache-2.0",
+  "scripts": {
+    "start": "vite",
+    "dev": "vite",
```

Comment:
I don't think we need an alias for this, since `dev` is the one used in the README.md I think we should take `start` out.